modules/libcpuid/0.6.5/patches/add_build_file.patch

```diff
@@ -0,0 +1,46 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,43 @@
++cc_library(
++  name = "libcpuid",
```

Comment:
The resulting library will be called `liblibcpuid`. Could you call the target `cpuid` and then add a public `alias` called `libcpuid`? That way the filename will be as expected and users can still depend on `@libcpuid`.

Reply:
thanks for the feedback, corrected!